docs/DISTROS.md

```diff
@@ -0,0 +1,170 @@
+# curl distros
+
+<!-- markdown-link-check-disable -->
+
+Lots of organizations distribute curl packages to end users. This is a
+collection of pointers to where to learn more about curl on and with each
+distro.
+
+We discuss curl distro issues, patches and collaboration on the [curl-distros
+mailing list](https://lists.haxx.se/mailman/listinfo/curl-distros).
+
```

Comment:
New entry:

## Alma Linux

- curl package source and patches: curl package source and patches
- curl issues: https://bugs.almalinux.org/view_all_bug_page.php *click Category and choose curl*
- curl security: https://errata.almalinux.org/ *search for curl*